docs/io.cozy.accounts.md

```diff
@@ -50,6 +50,23 @@ Generally, the konnector should be able to handle by itself this kind of relatio
 }
 ```
 
+### Vault Cipher
+
+An account can be synced in a password manager. In that case, the `vaultCipher` relationship can be used to store synchronization data.
+
+#### Example
+```json
+{
+  "relationships": {
+    "vaultCipher": {
+      "_id": "123abc",
+      "_doctype": "com.bitwarden.cipher"
```

Comment:
[nitpick] Can you add the missing comma to help github syntax highlighting?

Reply:
And it looks like we have use the plural form for doctypes. Can we use `com.bitwarden.ciphers` to be consistent?